core/frontend/src/components/autopilot/AutopilotUpdater.vue

```diff
@@ -0,0 +1,54 @@
+<template>
```

Comment:
**AutoPilotUpdater** sounds like a firmware update thing, or closer to it, can we change it to **AutopilotManagerData** or something closer to it.

Reply:
True. Maybe *AutoPilotDataUpdater*?

Reply:
This still implies on the data of the autopilot itself and not from the manager.. but that's not a problem that could hold the PR.
Maybe only **AutopilotManager** since it'll deal with everything related to it.

Reply:
Got it.
**AutoPilotManagerUpdater** so?

Reply:
:shipit: